src/Microsoft.DocAsCode.Common/Loggers/LogLevel.cs

```diff
@@ -8,6 +8,7 @@ public enum LogLevel
         Diagnostic = -1,
         Verbose,
         Info,
+        Suggestion,
```

Comment:
shall we keep the original order, i.e. putting `Suggestion` at the end of the list?

Reply:
Order here matters to make [`LogLevelThreshold`](https://github.com/dotnet/docfx/blob/a932308152152250273243968f1f5d161c23c0bd/src/Microsoft.DocAsCode.Common/Loggers/Logger.cs#L90) work.